recipes/libassert/all/conanfile.py

```diff
@@ -82,7 +84,14 @@ def generate(self):
         if is_msvc(self):
             tc.variables["USE_MSVC_RUNTIME_LIBRARY_DLL"] = not is_msvc_static_runtime(self)
 
-        if Version(self.version) >= Version("1.2.1"):
+        if Version(self.version) >= Version("2.0.0"):
+            # if not self.options.shared:
+                # tc.variables["LIBASSERT_STATIC_DEFINE"] = True
+            tc.variables["LIBASSERT_USE_EXTERNAL_CPPTRACE"] = True
+            tc.variables["LIBASSERT2"] = True
```

Comment:
I found I needed to do this to check in the test package whether to do `find_package(libassert)` or `find_package(assert)`. Not sure if there is a better solution.